tests/test_gp_interp.py

```diff
@@ -37,6 +37,30 @@
                       ('v0', float),
                       ('flux', float)])
 
+def get_correlation_length_matrix(correlation_length, g1, g2):
+    """
+    Produce correlation matrix to introduce anisotropy in kernel. 
+    Used same parametrization as shape measurement in weak-lensing 
+    because this is mathematicaly equivalent (anistropic kernel 
+    will have an elliptical shape).
+
+    :param correlation_length: Correlation lenght of the kernel.
+    :param g1, g2:             Shear applied to isotropic kernel.
+    """
+    if abs(g1)>1:
+        g1 = 0
+    if abs(g2)>1:
+        g2 = 0
+    e = np.sqrt(g1**2 + g2**2)
+    q = (1-e) / (1+e)
+    phi = 0.5 * np.arctan2(g2, g1)
+    rot = np.array([[np.cos(phi), np.sin(phi)],
+                    [-np.sin(phi), np.cos(phi)]])
+    ell = np.array([[correlation_length**2, 0],
+                    [0, (correlation_length * q)**2]])
+    L = np.dot(rot.T, ell.dot(rot))
```

Comment:
I guess this is normal that this is not symmetric, because before the rotation,  the matrix is within the bases of the ellipse axes. The version that you proposed is more readable, so I implemented it.